src/api/records.tsx

```diff
@@ -98,10 +98,38 @@ const fetchRecordCountQuery = (
 
 // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-constraint
 export const useRecords = <T extends unknown = Record[]>(
-  options?: UseQueryOptions<
+  options?: UseQueryOptions<Record[], AxiosError, T, [string]>
+): UseQueryResult<T, AxiosError> => {
+  useAppSelector(selectQueryParams);
```

Comment:
This theoretically will need search params like `dateRange`, but we can sort that out later as part of the Search epic

Reply:
Yes, I wasn't quite sure how to leave this, whether we should leave it in its generic state for later customisation later if necessary or if the whole thing should be reverted back. I decided to leave it until we had a better understanding. If it ain't broke, don't fix it

Reply:
Yeah, we can sort out params related to searching e.g. date ranges etc. for the Search epic as the date range is here because you implemented the search boxes UI a little early